amethyst_utils/src/ortho_camera.rs

```diff
@@ -0,0 +1,103 @@
+use amethyst_core::Axis2;
+use amethyst_core::cgmath::Ortho;
+use amethyst_core::specs::{Component, DenseVecStorage, Join, System, ReadExpect, ReadStorage, WriteStorage};
+use amethyst_renderer::{Camera, ScreenDimensions};
+
+/// `Component` attached to the camera's entity that allows automatically adjusting the camera's matrix according to preferences in the "mode" field.
```

Comment:
Could you explain it in layman's terms too? Like the way I did for the isometric stuff.